src/server/test/districts.controller.e2e-spec.ts

```diff
@@ -49,7 +49,27 @@ describe("DistrictsController", () => {
         .post("/api/districts/import/csv")
         .attach("file", `${__dirname}/data/de.csv`)
         .expect(200)
-        .expect([2, 1, 3]);
+        .expect({ districtsDefinition: [2, 1, 3] });
+    });
+  });
+
+  describe("import plan from block equivalency CSV with invalid block id in first row", () => {
+    it("should return a flag", () => {
+      return request(app.getHttpServer())
+        .post("/api/districts/import/csv")
+        .attach("file", `${__dirname}/data/bad_blockid.csv`)
+        .expect(200)
+        .expect({
+          districtsDefinition: [2, 1, 3],
+          rowFlags: [
+            {
+              rowNumber: 0,
+              errorText: "Invalid block ID",
+              rowValue: [100059900000024, 3],
```

Comment:
This should fix the tests:
```suggestion
              rowValue: ["100059900000024", "3"],
```